GeneralSQL/Error.py,sha256=kBrYZ17K0Jps9-DY3QFG_tLa9GOrLBJvoTLv8wwxfMk,2321
GeneralSQL/__init__.py,sha256=YECVqmS_N5Bqep7Y6zfMN9YvRj4LHonEduWgp4RN_fE,2119
GeneralSQL/Template/__init__.py,sha256=F80meUoOqQIOiYxE9ryCJzTMqFPBOwajkaf6Gk1e7Zk,1431
GeneralSQL/Template/base.py,sha256=3bFQygCmRR3GrpkoT83II0sOkmcEdlAP8zrFFWkkKzo,2994
GeneralSQL/db/Jmysql.py,sha256=Lbl5DqcouQ2OFejO8ODwWYQUIg5OYftEUed_jQBGr7o,16289
GeneralSQL/db/__init__.py,sha256=PgWm-6m5ZDVt2vjNOtrgO7sQYzXa6A0p8Y2RnoZNTl0,57
GeneralSQL-1.3.1.dist-info/METADATA,sha256=eeFT-BqgXBJ7NFx1uiuzdtdpIFSeWTS3YfzrE6M1yvw,42392
GeneralSQL-1.3.1.dist-info/WHEEL,sha256=GJ7t_kWBFywbagK5eo9IoUwLW6oyOeTKmQ-9iHFVNxQ,92
GeneralSQL-1.3.1.dist-info/top_level.txt,sha256=N28y8Tp5KNt4vFIdlg6UXzrY9QGtXwgK-oqRvdRBuh8,11
GeneralSQL-1.3.1.dist-info/RECORD,,
